misc.py

```diff
@@ -89,6 +89,9 @@ class Colors:
         which require the color to be generated at runtime.
     """
 
+    def __init__(self):
+        raise NotImplementedError("Colors is a static class, it cannot be instantiated")
```

Comment:
Surprised that this isn't the default when making an Enum class. What the hell python!

Reply:
Actually it defaults to `__call__` which accepts Foo values and returns a Foo object containing that value, so I guess I'll remove this.